release/verval/verval_test.go

```diff
@@ -0,0 +1,206 @@
+package main
```

Comment:
Are these tests included in the CCI config?

Reply:
I will check!

Reply:
Added to cci